test/acceptance/schema/get_schema_without_client_test.go

```diff
@@ -69,6 +69,9 @@ func testGetSchemaWithoutClient(t *testing.T) {
 						"trainingLimit": float64(100000),
 						"rescoreLimit":  float64(20),
 					},
+					"filteredSearch": map[string]interface{}{
```

Comment:
@abdelr  I think the term `filteredSearch` here is too generic and that we should change this to `acorn`. I understand it is more confusing what `acorn` does but we can enable by default at a later stage.

Additionally do we need a map here in that additional values will be needed or can we just use a `boolean`?